src/jit/utils.cpp

```diff
@@ -1194,6 +1194,9 @@ void HelperCallProperties::init()
 
         case CORINFO_HELP_NEW_CROSSCONTEXT:
         case CORINFO_HELP_NEWFAST:
+#ifdef FEATURE_READYTORUN_COMPILER
```

Comment:
Yes, I think that the code would be more readable and maintainable without these `#ifdef FEATURE_READYTORUN` that are not really making a lot of difference.